src/textual/message_pump.py

```diff
@@ -655,6 +658,16 @@ async def _dispatch_message(self, message: Message) -> None:
             # Allow apps to treat events and messages separately
             if isinstance(message, Event):
                 await self.on_event(message)
+            elif "debug" in self.app.features:
+                start = perf_counter()
+                await self._on_message(message)
+                if perf_counter() - start > SLOW_THRESHOLD_MS / 1000:
+                    log.warning.verbosity(message.verbose)(
+                        f"method=<{self.__class__.__name__}.{message.handler_name}>",
+                        ">>>",
```

Comment:
I don't think we need the ">>>", that was just to indicate a message being sent.